generators/dotnet-bots/templates/dotnet/bots/request-reply/Program.cs

```diff
@@ -1,50 +1,46 @@
-﻿using System;
-using System.IO;
 using apiClientDotNet.Models;
 using apiClientDotNet;
 using apiClientDotNet.Listeners;
 using apiClientDotNet.Services;
-using apiClientDotNet.Models.Events;
-using System.Diagnostics;
+using apiClientDotNet.Authentication;
+
 
 namespace RequestResponse
 {
     class Program
     {
         static void Main(string[] args)
         {
-            string filePath = Path.GetFullPath("config.json");
-            SymBotClient symBotClient = new SymBotClient();
-            DatafeedEventsService datafeedEventsService = new DatafeedEventsService();
-            SymConfig symConfig = symBotClient.initBot(filePath);
-            RoomListener botLogic = new BotLogic();
-            DatafeedClient datafeedClient = datafeedEventsService.init(symConfig);
-            Datafeed datafeed = datafeedEventsService.createDatafeed(symConfig, datafeedClient);
-            datafeedEventsService.addRoomListener(botLogic);
-            datafeedEventsService.getEventsFromDatafeed(symConfig, datafeed, datafeedClient);
+            SymConfig symConfig = new SymConfigLoader().loadFromFile("config.json");
+            SymBotRSAAuth botAuth = new SymBotRSAAuth(symConfig);
+            botAuth.authenticate();
+            SymBotClient botClient = SymBotClient.initBot(symConfig, botAuth);
+
+            // start listening for messages
+            DatafeedEventsService dataFeedService = botClient.getDatafeedEventsService();
+            MyIMListener listener = new MyIMListener(botClient);
+
+            dataFeedService.addIMListener(listener);
+            dataFeedService.getEventsFromDatafeed();
         }
     }
 
-    public class BotLogic : RoomListener
+    public class MyIMListener : IIMListener
```

Comment:
suggestion to rename to something like SimpleIMListener, DefaultIMListener etc. to "sounds" more professional :)

Reply:
ah yes completely overlooked that :-S
I will fix it